src/main/java/com/dezorganizacja/bridge/view/graphical/CreditsController.java

```diff
@@ -1,10 +1,110 @@
 package com.dezorganizacja.bridge.view.graphical;
 
+import com.dezorganizacja.bridge.presenter.MainPresenter;
+import javafx.animation.PathTransition;
+import javafx.animation.Timeline;
 import javafx.event.ActionEvent;
+import javafx.geometry.Insets;
+import javafx.scene.control.Label;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.*;
+import javafx.scene.text.Font;
+import javafx.util.Duration;
+
+import static java.lang.Thread.sleep;
 
 public class CreditsController extends Controller {
 
+    public AnchorPane creditsPane;
+
     public void closeCredits(ActionEvent actionEvent) {
         presenter.setState("main");
     }
+
+    @Override
+    public void init(MainPresenter presenter) {
+        super.init(presenter);
+
+        creditsPane.setBackground(new Background(new BackgroundFill(Color.BLACK, CornerRadii.EMPTY, Insets.EMPTY)));
+
+        int duration = 10000;
+
+        VBox myCreditsBox = new VBox();
+        myCreditsBox.getChildren().add((new CreditsNodeController()).setCodePieceLabel("OPs").addAuthor("Grzegorz Świrski").addAuthor("Marek Rusinowski").accept());
```

Comment:
Creditsy to jest zwykły VBox, który się przewija, można po prostu zaincludować fxmla wypełnionego danymi i go przewinąć. Dużo łatwiej by się go edytowało a w dodatku bez grzebania w kodzie